src/test/ui/consts/const-eval/panic-assoc-never-type.rs

```diff
@@ -13,4 +13,5 @@ impl PrintName {
 
```

Comment:
This file can now probably have its `build-fail` comment removed

Reply:
Removing the comment causes the test to fail with `error: ui test compiled successfully!`.

Reply:
As discussed on Zulip, the test will remain as is.